numpy/__init__.pyi

```diff
@@ -554,7 +564,11 @@ def empty_like(
     shape: Optional[_ShapeLike] = ...,
 ) -> ndarray: ...
 def full(
-    shape: _ShapeLike, fill_value: Any, dtype: DtypeLike = ..., order: str = ...
+    shape: _ShapeLike,
+    fill_value: Any,
+    dtype: DtypeLike = ...,
+    order: str = ...
+    like: ArrayLike = ...,
```

Comment:
```suggestion
    *,
    like: ArrayLike = ...,
```

Reply:
The asterix is still missing here.
Also it seems that a trailing comma is missing from, among others, `order: str = ...` (hence the typing-related CI failures).

Reply:
You're right, thanks for pointing it out. Should be fixed now (hopefully).

Reply:
Was this resolved? Also the `order: str = ...` has no trailing comma? Or am I missing something?

Reply:
The comma is there. This is resolved.

Reply:
Strange, must have been looking at some outdated diff. Thanks.